client/my-sites/plans/current-plan/thank-you-header/style.scss

```diff
@@ -0,0 +1,39 @@
+/** @format */
+.thank-you-header {
+	box-shadow: none;
+	text-align: center;
+
+	@include breakpoint( '>800px' ) {
+		font-size: 16px;
+		// padding: 50px 80px 30px;
+		margin-bottom: 40px;
+	}
+	@include breakpoint( '>1280px' ) {
+		padding: 50px 160px 30px;
+	}
+
+	.progress-bar {
+		max-width: 70%;
+		margin-bottom: 1em;
+	}
+}
+
+.thank-you-header__illustration {
+	display: none;
+	@include breakpoint( '>480px' ) {
+		display: inline;
+		width: 70%;
+	}
+	@include breakpoint( '>800px' ) {
+		width: 52%;
```

Comment:
Although these look good, the widths look made up. Any idea how we got to these particular values?

Reply:
Trail and error - seemed to look good was the criteria 🤷‍♂️

Reply:
I'm asking because these can get odd and inexplicable when we get to iterate or reuse this bit somewhere else. Back when I was working on Jetpack Onboarding in Calypso I remember we challenged some of the design mockups in favor of cleaner code and better consistency with what we already have in Calypso, and that only made it easier and cleaner as we iterated on the project.

Reply:
It depends - if we want it to be fluid, and resize as we resize the window, let's leave it as-is. If we want it to have a consistent size throughout between two breakpoints, we leave an absolute value as a `max-width`. This is working pretty smoothly currently, so I'm good with leaving it as-is if that's your preference.